core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -892,7 +892,7 @@ protected function generateGlobalButtons()
 
 			$v = \is_array($v) ? $v : array($v);
 			$label = \is_array($v['label']) ? $v['label'][0] : $v['label'];
-			$title = \is_array($v['label']) ? $v['label'][1] : $v['label'];
+			$title = \is_array($v['label']) ? ($v['label'][1] ?? '') : $v['label'];
```

Comment:
fix for extension see: https://github.com/oveleon/contao-component-style-manager/pull/46/files#diff-bc2c37cb346d11ba0a90db5779f412b8d9f79de1ed6a2a58451fe22a6efaf989R25

Reply:
Fixed in the extension see: https://github.com/oveleon/contao-component-style-manager/pull/46/commits/7f9110518fd05bd391c6b9829d99fda51df8f035#diff-5c38568b17b8589bfb3893db5658f251b4cf63594660bc5c0238d9d9d9d24271